src/main/java/hudson/plugins/locale/user/UserLocaleProperty.java

```diff
@@ -0,0 +1,58 @@
+package hudson.plugins.locale.user;
+
+import hudson.Extension;
+import hudson.model.User;
+import hudson.model.UserProperty;
+import hudson.model.UserPropertyDescriptor;
+import hudson.plugins.locale.Messages;
+import hudson.plugins.locale.PluginImpl;
+import net.sf.json.JSONObject;
+import org.jvnet.localizer.LocaleProvider;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.StaplerRequest;
+
+import java.util.Locale;
+
+public class UserLocaleProperty extends UserProperty {
+    private String localeCode;
+
+    private Locale locale; // for the cache purpose
+
+    public UserLocaleProperty(String localeCode) {
+        this.localeCode = localeCode;
+    }
+
+    public Locale getLocale() {
+        return locale;
+    }
+
+    public String getLocaleCode() {
+        return localeCode;
+    }
+
+    @DataBoundSetter
+    public void setLocaleCode(String localeCode) {
+        try {
+            locale = PluginImpl.parse(localeCode);
+        } catch (IllegalArgumentException e) {
+            // ignore this exception
+        }
+        this.localeCode = localeCode;
+    }
+
+    @Extension
```

Comment:
Add a `userLocale()` symbol to simplify  configuration via JCasC? We cannot use `locale` due to potential conflicts with the global properties. CC @timja